client/pages/sections/protocols/sections.js

```diff
@@ -112,25 +113,46 @@ const sortGranted = (sections, isGranted) => (a, b) => {
   return sections[a].granted.order - sections[b].granted.order;
 }
 
-const ProtocolSections = ({ sections, protocolState, editable, newComments, ...props }) => (
+const changedBadge = change => {
+  switch (change) {
+    case LATEST:
+      return <span className="badge changed">changed</span>;
+    case GRANTED:
+      return <span className="badge">amended</span>;
+    default:
+      return null;
+  }
+}
+
+const ProtocolSections = ({ sections, protocolState, editable, newComments, ...props }) =>  {
+  return (
   <Accordion open={getOpenSection(protocolState, editable, sections)} toggleAll={!props.pdf}>
     {
       Object.keys(sections).sort(sortGranted(sections, props.isGranted)).filter(section => !sections[section].show || sections[section].show(props)).map((section, sectionIndex) => (
+        <Fragment key={section}>
+        {
+          changedBadge(changed(sections[section].fields.map(field => field.name), props.latest, props.granted))
+        }
         <ExpandingPanel alwaysUpdate={section === 'conditions' || section === 'authorisations'} key={section} title={getTitle(sections[section], newComments, props.values)}>
           {
             getSection(section, { ...props, protocolState, editable, ...sections[section], sectionsLength: size(sections), sectionIndex })
           }
         </ExpandingPanel>
+        </Fragment>
```

Comment:
The indentation here is out of whack somehow. There shouldn't be two closing tags at the same level.